apps/nextjs-website/src/components/organisms/ChatbotProvider/ChatbotProvider.tsx

```diff
@@ -1,7 +1,7 @@
 'use client';
 
 import { useUser } from '@/helpers/user.helper';
-import { FC, PropsWithChildren } from 'react';
+import { FC, PropsWithChildren, useState } from 'react';
```

Comment:
This import is not used
```suggestion
import { FC, PropsWithChildren } from 'react';
```